src/Qowaiv/Identifiers/Id.cs

```diff
@@ -240,6 +241,24 @@ void IXmlSerializable.WriteXml(XmlWriter writer)
     /// <summary>Casts the <see cref="Uuid"/> to an identifier.</summary>
     public static explicit operator Id<TIdentifier>(Uuid id) => Create(id);
 
+    /// <summary>Converts the <see cref="string"/> to <see cref="Id{TIdentifier}"/>.</summary>
+    /// <param name="s">
+    /// A string containing the identifier to convert.
+    /// </param>
+    /// <param name="provider">
+    /// The format provider.
+    /// </param>
+    /// <returns>
+    /// <returns>
+    /// The parsed identifier.
+    /// </returns>
+    /// <exception cref="FormatException">
+    /// <paramref name="s"/> is not in the correct format.
+    /// </exception>
+    [Pure]
+    public static Id<TIdentifier> Parse(string s, IFormatProvider? provider)
```

Comment:
If IParsable is only for net7 or greater, should these methods also be inside an if?

Reply:
For the other SVO's (where this is generated) those overloads are also available for all targets.